coverage.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+forge install
+find lib/aave-v3-core/contracts -type f -name "*.sol" -exec sed -i 's/pragma solidity 0\.8\.10;/pragma solidity 0.8.13;/g' {} +
```

Comment:
`sed` behaves differently across platforms, so we might want to use another command here (like `awk`) to ensure it works for all users. For example for me on macOS this coverage script fails:

```
sed: 1: "./lib/aave-v3-core/cont ...": invalid command code .
```

Reply:
https://github.com/ScopeLift/flexible-voting-clients/pull/1/commits/4dfff53f6f96accfc59fca2b31d115e0b2e161b0